### PR TITLE
Add PayPal GraphQL support

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) documents the PaypalGraphQl module. 

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/paypal.html (doesn't exist yet)
- ...

## Links to Magento source code

- Not applicable. Internal repo
- ...

### CODE REVIEW

File: guides/v2.3/graphql/reference/paypal.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+The `Paypal` endpoint provides support for the following PayPal payment methods:
+
+* PayPal Express Checkout
+* PayPal Payflow Pro and PayPal Payflow Link
+
+The following steps describe the flow of calls required to complete a typical PayPal checkout transaction. A successful purchase requires that you send three mutations to PayPal, and the buyer must approve the purchase by logging in to PayPal.
+
+1. **Send a token request.** When the buyer clicks a PayPal button, the frontend executes the `createPaypalExpressToken` mutation. The Magento `PaypalGraphQl` module gathers information in the specified cart and sends this information to PayPal.
+
+2. **PayPal sends a token.** If the token request succeeds, PayPal returns a token and a payer ID. PayPal also sends payment-related data that is outside the scope of GraphQL. You must include this token in subsequent steps. The buyer is redirected to the payment confirmation page, which was specified in the token request.
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal.md]: "PayPal returns a token" as heading for this section?

File: guides/v2.3/graphql/reference/paypal.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+6. **Complete the transaction.** Place the order with the `placeOrder` mutation. PayPal captures the payment by transferring the funds from the customer account to the appropriate merchant account. Magento creates an order, ready for fulfillment.
+
+## Mutation
+
+The `createPaypalExpressToken` mutation initiates a PayPal checkout transaction and receives a token. You must specify this token when setting a PayPal payment method.
+
+## Syntax
+
+`createPaypalExpressToken(input: PaypalExpressTokenInput): PaypalExpressToken`
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal.md]: Other mutations did this syntax formatting:
mutation: {createCustomer(input: CustomerInput!) {CustomerOutput}}

has this convention changed or are the PayPal mutations different?

File: guides/v2.3/graphql/reference/paypal.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+## Examples
+
+These examples show all the mutations required to complete a PayPal purchase.
+
+### Request a PayPal token
+
+The PayPal `token` will be used in the other mutations. The raw response from PayPal also includes the Payer ID. Extract the payer ID so that it used in the mutation that sets the payment method.
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal.md]: Payer ID vs. payer ID?

File: guides/v2.3/graphql/reference/paypal.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+    }
+  }
+}
+```
+
+## Input attributes
+
+### PaypalExpressTokenInput {#PaypalExpressTokenInput}
+
+The `PaypalExpressTokenInput` object defines the attributes required to receive a payment token from PayPal
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal.md]: missing ending period.

File: guides/v2.3/graphql/reference/paypal.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cart_id` | String! | The unique ID that identifies the customer's cart
+`code` | String! | Payment method code
+`express_button`: | Boolean | Indicates whether the buyer selected the PayPal Express checkout button. The default value is `false`
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal.md]: In other places it's: PayPal Express Checkout (all initial cap'd).

File: guides/v2.3/graphql/reference/paypal.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+`urls` | [`PaypalExpressUrlsInput`](#PaypalExpressUrlsInput) | Defines a set of URLs to redirect to in response to the token request
+`use_paypal_credit` | Boolean | Indicates whether the buyer clicked the Paypal credit button. The default value is `false`
+
+### PaypalExpressUrlsInput {#PaypalExpressUrlsInput}
+
+The `PaypalExpressUrlsInput` object contains a set of URLs that PayPal uses to respond to a token request.
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cancel_url` | String! | The URL of the original page on your website where the buyer initially chose PayPal as a payment type
```
Comment (erikmarr) [inline guides/v2.3/graphql/reference/paypal.md]: Just wanting to confirm that this description is correct for `cancel_url`? I would expect to see something about canceling...but maybe this is a Web term I'm not familiar with.

File: guides/v2.3/graphql/reference/paypal.md (hunk 2 above)
Comment (keharper) [reply guides/v2.3/graphql/reference/paypal.md]: It's different

File: guides/v2.3/graphql/reference/paypal.md (hunk 6 above)
Comment (keharper) [reply guides/v2.3/graphql/reference/paypal.md]: Clarified.

**Pull request merged**